tests/ui/map_err.stderr

```diff
@@ -0,0 +1,10 @@
+error: `map_err` has thrown away the original error
+  --> $DIR/map_err.rs:21:32
+   |
+LL |     println!("{:?}", x.map_err(|_| Errors::Ignored));
+   |                                ^^^ help: Allow the error enum to encapsulate the original error: `|e| Errors::Ignored(e)`
```

Comment:
I think we might want to hold off on suggesting how they fix it, since its so circumstantial.

Reply:
I update this, please take a look at your earliest convenience 😄